### ISSUE TITLE
Create Lightning Invoices

### ISSUE THREAD

**Issue by johncantrell97** [LN-Juggernaut/juggernaut-desktop#39]
We need a screen for someone to generate a lightning invoice.  Will need to set amount of satoshis and optional memo as a starting point.

Once the invoice is generated we should display information about the invoice including a qr code and a way to copy invoice to clipboard.

**Comment by stefan-mihaila** [comment]
Where do you think we should add this, payment, and settings? I was thinking to have a ConversationListFooter and have a few actions there that we want accessible with one click (without going to the menu in the ConversationListHeader)

**Comment by johncantrell97** [comment]
Yes, I'm still unsure about a lot of the final UI for Juggernaut.  I think we will definitely end up having a ConversationListFooter as you mention (similar to telegram).  I think we will also end up having a LeftSidebarNav similar to Slack.  My gut is that the LeftSidebarNav will be for toggling between core "apps" of Juggernaut (Messenger, Wallet, <Feature Z>).

If we end up doing something like that where the wallet is more separated from the messenger then it would probably not make sense to put the create invoice in the footer of the messenger. 

So in general I've just defaulted to tossing everything into a menu for now until the next iteration of the UI makes sense.  My gut would be to continue to do the same for this feature unless you really just want to explore the footer concept.
